draft-ietf-jsonpath-base.md

```diff
@@ -541,6 +541,16 @@ The well-formedness and the validity of JSONPath queries are independent of
 the JSON value the query is applied to. No further errors relating to the
 well-formedness and the validity of a JSONPath query can be
 raised during application of the query to a value.
+This clearly separates well-formedness/validity errors in the query
+from mismatches that may actually stem from flaws in the data (see
+also {{data-attack}}).
+
+Often, mismatches between the structure expected by a valid query
```

Comment:
Suggest: It is possible that mismatches … in the data lead to empty query results.

(Don't think "often" is called for, any path to replace it is OK by me)

Reply:
I went for a simple "can".